src/views/Notifications/ProjectJoin/index.js

```diff
@@ -154,13 +179,17 @@ export default class ProjectJoinItem extends React.PureComponent {
             requestedBy,
             respondedBy,
             status,
+            role,
         } = data.details;
 
-        const approvalText = status === 'rejected' ? 'rejectedText' : 'acceptedText';
+        let approvalText = 'rejectedText';
+        if (status === 'accepted') {
```

Comment:
Never use variable to get string identifier. Use two _ts methods

Reply:
Fixed